src/nxt_php_sapi.c

```diff
@@ -411,10 +422,63 @@ nxt_php_setup(nxt_task_t *task, nxt_process_t *process,
         nxt_php_set_options(task, value, ZEND_INI_USER);
     }
 
+    zend_hash_init(&nxt_php_user_config_cache, 0, NULL, (dtor_func_t) nxt_php_user_config_cache_entry_dtor, 1);
+
     return NXT_OK;
 }
 
 
+static void
+nxt_php_activate_user_config(nxt_php_run_ctx_t ctx) {
```

Comment:
Is it required to pass `ctx` by value, not the pointer to it?
Function start brace should be on a new line.